lib/tasks/gir.thor

```diff
@@ -21,19 +21,25 @@ class GirCLI < Thor
     end
     glob.each do |path|
       puts 'Generating scraper for ' + File.basename(path) + '...'
+      begin
```

Comment:
Indentation is a bit weird in this block ... is it somehow mixing tabs and spaces?